src/classes/LineItemTriggerHandler.cls

```diff
@@ -12,6 +12,7 @@ public with sharing class LineItemTriggerHandler {
 		Set<Id> orderIds = new Set<Id>();
 		for (Line_Item__c i : items) {
 			orderIds.add(i.Order__c);
+            update i;
```

Comment:
This DML statement is not surrounded by any try-catch block. Please add one to ensure exceptions are handled properly.